funcs_scripts.php

```diff
@@ -216,6 +247,17 @@ function is_gha_repository()
     return !is_module() && strpos($MODULE_DIR, '/gha-') !== false;
 }
 
+/**
+ * Determine if the module being processed should include a dispatch-ci.yml file
+ *
+ * Example usage:
+ * should_use_dispatch_ci()
+ */
+function should_use_dispatch_ci()
+{
+    return (is_module() || is_composer_plugin()) && !is_docs() && !is_gha_repository();
+}
```

Comment:
Unless you can think of other times we'd use this, this logic should just be put directly into the `dispatch-ci.php` script. Otherwise every script would have it's own function in here.

Reply:
Updated